### ISSUE TITLE
how to use this? 

### ISSUE THREAD

**Issue by Necko1** [WesJD/AnvilGUI#322]
lol I didn’t even have time to start writing code, and I already have some kind of problem.
![изображение](https://github.com/WesJD/AnvilGUI/assets/128916594/d6631c33-391b-4585-8244-fb7d1d87d74d)
I also tried to change the paper api to spigot api just in case but nothing has changed
also i tried to add it via jar file. I was able to code something, but then again there was an error.
![изображение](https://github.com/WesJD/AnvilGUI/assets/128916594/2bbd3bf1-576a-47a2-9e5b-160df3ace2e9)
I probably don't understand something, can you help?

**Comment by portlek** [comment]
did you add the snapshot repository in the README?

**Comment by Necko1** [comment]
After so much time, trial and error, I came to this list. guys who use intellij idea will get the same error, repeat:
1. download the latest build of the anvilgui jar file (https://ci.codemc.io/job/WesJD/job/AnvilGUI/lastBuild/)
2. add this jar file to the library.
3. DELETE THE .idea/artifacts FOLDER
4. go to pom.xml and shade library, as written in README.md
5. add artifacts back in project structure and build the plugin.
That's all! and I came to this after an hour and a half btw.
